Fix/doc 4288 (#99)
exoplatform/exo-documentation: docs/Administration.rst

- Visibility status : default visibility option (visible, secret) for the space
- Registration : default registration policy for the space (open, closed, validation)
- Applications : list of applications populated by default in the space
- Permissions : list of groups/roles that are authorized to create a space from this template
- Banner View : the image that will be used by default as banner for the space (click to display)

.. note:: The Manage Template screen currently only allows to displaying all templates informations. Customising a space template must be done by editing config files as explained in :ref:`developers guide <PLFDevGuide.SpaceTemplates.Extend>` 

>           To display the banner, you can simply click on the eye icon so that the banner will be displayed, and you can click outside the banner or on ESC to close it.

Reviewer: ```suggestion
          To display the banner, simply click on the eye icon. To close it, click outside the banner or on ESC button of the keyboard.
```
